### DISCUSSION TITLE
Overall leaderboard?

### DISCUSSION THREAD

**Discussion by dietmarwo** [esa/SpOC2#21] [Competition Q&A]
Is there or will there be an overall leaderboard for all 3 competitions summing up the scores for each team
(10 points for 1th, 9 points for 2nd, ...) ? This could encourage teams to engage in the other competitions. 
We don't have much uploads so far. This means, even a weak score helps with the current overall rank. As it is
now, teams may hold back their solutions to not give away too much information to the competitors. Or are not motivated enough to work on all three competitions.

**Comment by DreamTlaloc** [comment]
Hi Dietmar,

I can't speak for a leaderboard on the website since I wouldn't be the one who would be making it, but on the second point; we strongly encourage teams to submit their best solutions more or less when they find them, and not to hide solutions until just before the deadline, or try any other ways to metagame the competition. While we want people to be competitive, SpOC is ultimately about having fun and exploring unusual problems, which is only enriched by a lively struggle on the leaderboards.

Thanks,
Amy

**Reply by dietmarwo** [reply]
>SpOC is ultimately about having fun and exploring unusual problems, which is only enriched by a lively struggle on the leaderboards

Fully agree. This is exactly my "strategy": Uploading solutions immediately to encourage others to improve. It is just that an overall leaderboard would provide additional motivation to follow this rule. Even a "very bad" upload would contribute to the overall score, as long as there are less than 10 uploads for a competition - as it is the case now for all three of them.

**Reply by darioizzo** [reply]
Unfortunately it is not possible to provide such a leaderboard in the ESA optimize platform since we designed it for different types of competition formats, and not for SPOC specifically. We will provide such a leaderboard after the competition ends (and maybe a preliminary one at half way), but using other channels.

**Reply by dietmarwo** [reply]
>and maybe a preliminary one at half way

That would be great, this is less than 2 weeks from now.

**Reply by dietmarwo** [reply]
>Unfortunately it is not possible to provide such a leaderboard in the ESA optimize platform since we designed it for different types of >competition formats, and not for SPOC specifically.

Perhaps there's an alternative method to encourage teams to take part in various challenges that is solely centered around the Optimize platform. Currently, the ESA Optimize platform, found at https://optimise.esa.int/challenges, displays six challenges, with plans for additional ones in the future.

Consider the concept of a comprehensive leaderboard. It would be akin to the existing SPOC ranking system, awarding 10 points for first place, 9 points for second, and so forth. However, rather than being confined to SPOC competitions, it would encompass all challenges hosted on the ESA Optimize platform.

Such a leaderboard could serve to:

- Encourage teams to continue submitting enhancements even after the close of a competition deadline.
- Spur teams to participate in a diverse array of challenges.

The ESA Optimize platform already allows for perpetual submissions, a pioneering concept in this field. If teams are learning from one another, exchanging insights at workshops or through reading workshop presentations, we should celebrate and document this progress with improvements in the leaderboard rankings.

Implementing a comprehensive leaderboard could help us further leverage this innovative idea.
